doc/api/cli.md

```diff
@@ -272,6 +272,13 @@ reference. Code may break under this flag.
 `--require` runs prior to freezing intrinsics in order to allow polyfills to
 be added.
 
+### `--fs-url-strings`
```

Comment:
maybe make this `--experimental-fs-url-strings` to make it a bit clearer that this is an experimental feature intended to be default behavior?